src/scripts/main.js

```diff
@@ -0,0 +1,10 @@
+const loadComponents = async (id, url) => {
+    const res = await fetch(url);
+    const text = await res.text();
+    return (document.getElementById(id).innerHTML = text);
+  };
+
+  window.onload = () => {
+    loadComponents("hero", "./src/sections/hero.html");
+
```

Comment:
Hello, @Algor-Fernand since you create a new script file Please don't also forget to load the manage team component.
`loadComponents("manage", "./src/sections/manage-team.html");`